dist/newrelic-angular.js

```diff
@@ -8,6 +8,76 @@
   ]);
 
   'use strict';
+  (function(angular) {
+    
+    angular.module('newrelic-angular.interceptor.http', [])
+      .provider('httpInterceptor', HttpInterceptorProvider)
+      .config(HttpInterceptorConfig);
+      
+    /**
+     * Http Interceptor
+     * Intercepts HTTP response and request errors and logs them to new relic
+     * @param {Promise} $q
+     * @param {Object} $log
+     * @return Object
+     */
+    
+    function HttpInterceptorProvider() {
+       
+      var statusesToIgnore = [];
+      
+      this.setStatusesToIgnore = setStatusesToIgnore;
+      
+      function setStatusesToIgnore(statuses) {
+        statusesToIgnore = statuses || [];
+      }
+      
+      function shouldIgnoreStatus(status) {
+        status = +status;
```

Comment:
I think a better pattern would be `angular.isNumber` or `isFinite`. It may make this a bit easier to understand (it took me a minute to figure out what you were doing here). 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite